### ISSUE TITLE
Parameter setting issue

### ISSUE THREAD

**Issue by Accept111** [Jaron-U/conditional_diffusion_compression#2]
Thank you for your work on this repository. It has been incredibly helpful for my project!
I checked the parameter settings in the original codebase and found that they were not very clear. Could the author please provide a complete set of parameter settings for the pre-warming and training stages?
This will be very helpful to me. Thank you again to the author for making the source code open！

**Comment by Jaron-U** [comment]
Hi, I have modified the training parameters directly into `train.py` and have uploaded it. But I still recommend you to check out the original codebase.

**Comment by Accept111** [comment]
Hello, author!
1、I wonder if I need to warm up the model every time I make changes to the model architecture.
2、In addition to the learning rate, in addition to β=1e-6 and training step length =700000, the other parameters are the same as the original code base, I will use the parameters posted out, please the author to help me look at it.
![image](https://github.com/user-attachments/assets/7bbdef2a-4b74-4782-a0c7-8c6bbd65dee9)

Thank you!!

**Comment by Jaron-U** [comment]
是的，都需要warm up, 但我强烈建议你看看原论文，或者直接问原仓库作者，他回答issue还挺快的。

**Comment by Accept111** [comment]
好的，感谢作者回答！
